src/component/handlers/edit/editOnInput.js

```diff
@@ -88,8 +89,41 @@ function editOnInput(editor: DraftEditor): void {
   var domSelection = global.getSelection();
 
   var {anchorNode} = domSelection;
-  if (anchorNode.nodeType !== Node.TEXT_NODE) {
-    return;
+  const isNotTextNode =
```

Comment:
Do we understand enough about this section to confidently merge? (For now, flagging this for further personal investigation)